doc/features.md

```diff
@@ -0,0 +1,74 @@
+Network Module Features
+-----------------------
+
+Network Services
+======================
+
+- wicked
+- Network Manager
+- no network service
+
+IPv6
+==============
+
+- enable/disable it
+
+DHCP Client options
+====================
+
+- set hostname according to dhcp response
+- set machine identifier for asking dhcp
+- allow to set specific option for hostname to send for dhcpd client ( how often it is used? )
+
+DNS
+===================
+
+- set hostname
+- set domain
+- change hostname with dhcp ( looks same as in dhcp client options )
+- assign hostname to local ip ( so own hostname is always resolvable )
+- modify /etc/resolv.conf, keep it manage automatic or modify it manually with selected values
+
+Routing
+=========
+
+- allows to assign gateway for ipv4 and ipv6 and also assign via which device it can be reached
+- allows to enable ipv4 and ipv6 forwarding
+- allows manual edit of routing table
+
+
+Devices
+=======
+
+- allows to setup it without setup (for bondings), dhcp or static ( with ip, hostname, mask and iBFT )
```

Comment:
ehm no. That note about iBFT belongs to "None" configuration - it means that configuration should be read from firmware of iBFT aware device